etc/config/csharp.defaults.properties

```diff
@@ -1,14 +1,21 @@
-compilers=dotnet6csharp
+compilers=dotnet6csharp:dotnet7csharp
 supportsBinary=false
 needsMulti=false
 compilerType=csharp
-defaultCompiler=dotnet6csharp
+defaultCompiler=dotnet7csharp
 
 compiler.dotnet6csharp.exe=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
 compiler.dotnet6csharp.name=.NET 6.0.101
 compiler.dotnet6csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked
-compiler.dotnet6csharp.runtimeId=linux-x64
 compiler.dotnet6csharp.targetFramework=net6.0
+compiler.dotnet6csharp.sdkVersion=6.0.0
 compiler.dotnet6csharp.buildConfig=Release
-compiler.dotnet6csharp.additionalSources=
 compiler.dotnet6csharp.langVersion=latest
+
+compiler.dotnet7csharp.exe=/opt/compiler-explorer/dotnet-v7.0.0/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
+compiler.dotnet7csharp.name=.NET 7.0.100
+compiler.dotnet7csharp.clrDir=/opt/compiler-explorer/dotnet-v7.0.0/bin/coreclr/Linux.x64.Checked
+compiler.dotnet7csharp.targetFramework=net7.0
+compiler.dotnet7csharp.sdkVersion=7.0.100-rc.1.22431.11
```

Comment:
The sdk version should match here: https://github.com/dotnet/runtime/blob/d099f075e45d2aa6007a22b71b45a08758559f80/global.json#L3